src/ReduxStore/selectors/matchedCitiesSelectors.js

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from 'reselect';
+
+export const getMatchedCitiesData = state => state.matchedCities.cities;
+
+export const transformCitiesData = createSelector(getMatchedCitiesData, cities => {
+  return [...cities]
```

Comment:
you can do the mapping first so that you won't have to additionally create new array